validphys2/src/validphys/scripts/vp_unweight.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
+import pandas as pd
+from scipy.special import xlogy
+from typing import Tuple
+from tqdm import tqdm
```

Comment:
I'm not too fond of tqdm because it usually makes logs and debugging harder to read.
But if you are set on using it you must add it to the dependencies as well.